### PR TITLE
[XBiome] Fixed issues for setting biomes on 1.16+

### PR DESCRIPTION
**Pull request by IridiumLLC**
In the Nether update, they introduced the ability to have different biomes depending on the Y axies. This will fix setting the biome on 1.16+ versions

### CODE REVIEW

File: src/main/java/com/cryptomorin/xseries/XBiome.java [hunk 1]
```diff
@@ -167,7 +168,8 @@ private static String format(@Nonnull String name) {
         for (int i = 0; i < len; i++) {
             char ch = name.charAt(i);
 
-            if (!appendUnderline && count != 0 && (ch == '-' || ch == ' ' || ch == '_') && chs[count] != '_') appendUnderline = true;
+            if (!appendUnderline && count != 0 && (ch == '-' || ch == ' ' || ch == '_') && chs[count] != '_')
+                appendUnderline = true;
```
Comment (CryptoMorin) [inline src/main/java/com/cryptomorin/xseries/XBiome.java:172]: Please change this line back.

File: src/main/java/com/cryptomorin/xseries/XBiome.java [hunk 2]
```diff
@@ -240,9 +242,14 @@ public Biome getBiome() {
         // Apparently setBiome is thread-safe.
         return CompletableFuture.runAsync(() -> {
             for (int x = 0; x < 16; x++) {
-                for (int z = 0; z < 16; z++) {
-                    Block block = chunk.getBlock(x, 0, z);
-                    if (block.getBiome() != biome) block.setBiome(biome);
+                // y loop for 1.16+ support (vertical biomes).
+                // As of now increasing it by 4 seems to work.
+                // This should be the minimal size of the vertical biomes.
```
Comment (CryptoMorin) [inline src/main/java/com/cryptomorin/xseries/XBiome.java:247]: > This should be the minimal size of the vertical biomes.

Can you also reference the source that why it should be specifically 4?

Comment (CryptoMorin) [reply src/main/java/com/cryptomorin/xseries/XBiome.java:247]: It'd be even better if there's a source available from either Spigot or Minecraft that explains the biome system in 1.16

**Review by CryptoMorin — CHANGES_REQUESTED** [review]

File: src/main/java/com/cryptomorin/xseries/XBiome.java (hunk 2 above)
Comment (IridiumLLC) [reply src/main/java/com/cryptomorin/xseries/XBiome.java:247]: > > This should be the minimal size of the vertical biomes.
> 
> Can you also reference the source that why it should be specifically 4?

Trial and error, 4 seemed to be the biggest number before issues started occuring :shrug:

Comment (CryptoMorin) [reply src/main/java/com/cryptomorin/xseries/XBiome.java:245]: Ah ok, I see. Thanks

**Pull request merged**